backend/spec/rails_helper.rb

```diff
@@ -43,4 +43,13 @@
       ActiveJob::Base.queue_adapter.performed_jobs.clear
     end
   end
+
+  config.before do
+    # this is so we don't hit the Translation API in tests, don't know of a civilised way to disable it
+    @translate_content_double = instance_double(Translations::TranslateContent)
```

Comment:
I think more elegant solution is either use webmocks (and then stub request) or use `vcr` which makes it even simpler. Also this stubbing should be turned on only at tests which it apply for --> in this case, it will be done for all tests, which I think is kind of overkill

Reply:
OK, good point